nuxt.config.js

```diff
@@ -48,7 +48,23 @@ export default {
     },
     // Global page headers (https://go.nuxtjs.dev/config-head)
     // Move to layout/default.vue due to gtm-module not support head function (https://github.com/nuxt-community/gtm-module/issues/56)
-    head: {},
+    head: {
+        link: [
```

Comment:
is this necessary? we can apply the class `font-serif` since we have this tailwind config.

https://github.com/pycontw/pycontw-frontend/blob/83ec5814511ab46c5fdb5379f2241dfabd605cab/tailwind.config.js#L16

Reply:
[b4c47bb](https://github.com/pycontw/pycontw-frontend/pull/259/commits/b4c47bb54b8e2903ce43676399d7e9079cab2a57)

Reply:
e11e82f